orthoinference/src/main/java/org/reactome/orthoinference/EventsInferrer.java

```diff
@@ -148,6 +150,7 @@ public static void inferEvents(Properties props, String pathToConfig, String spe
 			return;
 		}
 		String humanInstanceDbId = sourceSpeciesInst.iterator().next().getDBID().toString();
+		orthologousPathwayDiagramGenerator = new OrthologousPathwayDiagramGenerator(dbAdaptor, dbAdaptorPrev, speciesInst, personId, Long.valueOf(humanInstanceDbId));
```

Comment:
Is the `humanInstanceDbId` needed as specifically a String later on?  If not, maybe it would be better to have it be a `long` by removing the call to `.toString()` in the line before this and removing the need to parse it to a long value in this line.

Reply:
Yep good point